filesystem/PKGBUILD

```diff
@@ -6,6 +6,7 @@
 pkgname=filesystem
 pkgver=2018.02
 pkgrel=2
+epoch=1
```

Comment:
Git for Windows does not modify `filesystem` directly. Therefore this change should be unnecessary.